websites/M/MonBureauNumérique/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "902671568856047636",
+		"name": "orion_off"
+	},
+	"service": "MonBureauNumérique",
+	"description": {
+		"en": "MonBureauNumerique is a digital working environment designed by Skolengo and used by students in the Grand-Est region of France.",
+		"fr": "MonBureauNumerique est un environnement de travail numérique conçu par Skolengo et utilisé par les élèves de la région Grand-Est de France."
+	},
+	"url": "www.monbureaunumerique.fr",
+	"regExp": "\\b[a-zA-Z0-9.-]+\\.monbureaunumerique\\.fr\\b",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/xrb5WeX.png",
```

Comment:
[Logos](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) must be 512x512px

Reply:
Oops ! I will correct this !